chains/solana/contracts/programs/ccip-router/src/instructions/v1/fee_quoter.rs

```diff
@@ -38,7 +38,7 @@ pub fn fee_for_msg(
     fee_token_config: &BillingTokenConfig,
     additional_token_configs: &[Option<BillingTokenConfig>],
     additional_token_configs_for_dest_chain: &[PerChainPerTokenConfig],
-) -> Result<SVMTokenAmount> {
+) -> Result<(SVMTokenAmount, Vec<u8>, bool)> {
```

Comment:
Can the result be a struct instead of a tuple? It will be more easy to read and follow. Also, can you add docs here explaining what the response of this message?